README.md

```diff
@@ -65,7 +215,20 @@ gsutil -m cp ./*Nov*.txt gs://httparchive/crawls_manifest/
 - Dataflow temporary and staging artifacts in GCS
 - BigQuery (final landing zone)
 
-## Temp table cleanup
+## Builds and Deployments
```

Comment:
It'd also be useful to document the config files like `Dockerfile` and `*.yaml`

Reply:
Updated! Please have a look.